FusionMCPBridge/handlers/manufacture/setups/setup.py

```
@@ -0,0 +1,2178 @@
+    except Exception as e:
+        return {
+            "mode": "unknown",
+            "dimensions": {"length": 0, "width": 0, "height": 0, "diameter": 0},
+            "position": [0, 0, 0],
+            "material": "unknown",
+            "geometry_id": None,
+            "error": f"Error extracting stock info: {str(e)}"
+        }
+
+
+def _extract_model_reference(setup) -> str:
```

Ruff (0.14.10)

Unused function argument: `setup`

(ARG001)